File: Unidad 3/C#/Programa_19.cs
using System;
namespace Programa_19;
class Si{
    static void Main(string[] args){
        double num;
        int c, cero=0, mas=0, menos=0;

        Console.Write("Cantidad de numeros a evaluar: ");
        c=Convert.ToInt32(Console.ReadLine());
        
        for(int i=1;i<=c;i++){
            Console.Write("Ingrese un numero: ");
            num=Convert.ToDouble(Console.ReadLine());
            if(num==0){
                cero++;
            }else if(num>0){
                mas++;
            }else{
                menos++;
            }
        }
        Console.WriteLine("Cantidad de numeros menores a 0: "+menos);
        Console.WriteLine("Cantidad de numeros igaules a 0: "+cero);
        Console.WriteLine("Cantidad de numeros mayores a 0: "+mas);

    }
}

File: Unidad 3/Java/Programa_19.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.programa_19;
import java.util.Scanner;
/**
 *
 * @author gaelr
 */
public class Programa_19 {

    public static void main(String[] args) {
       Scanner sc = new Scanner(System.in);
       double num; 
       int c, cero=0, mas=0, menos=0;
       
       System.err.print("Cantidad de numeros a evaluar: ");
       c=sc.nextInt();
       
       for(int i=1;i<=c;i++){
           System.err.print("Ingrese un numero: ");
           num=sc.nextDouble();
           if(num==0){
               cero++;
           }else if(num>0){
               mas++;
           }else{
               menos++;
           }
       }
       System.out.println("Cantidad de numeros menores a 0: "+menos);
       System.out.println("Cantidad de numeros igaules a 0: "+cero);
       System.out.println("Cantidad de numeros mayores a 0: "+mas);
       
       
       
    }
}
